app/views/content/life-as-a-teacher/teaching-as-a-career/_categories.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <section class="category col-space-m">
+    <section class="container category__cards main-section">
+      <nav class="category__nav-cards">
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Change to a career in teaching",
```

Comment:
I wonder about the order of this?

ECT info is relevant to everyone - career changers info (which is a bit weak currently) is not

So I wonder if it makes more sense to be ECT / Insights / Career changers for now?

(Appreciate that there will be new content to come which would then go at the top of the page!)